sharktank/tests/evaluate/perplexity_iree_test.py

```diff
@@ -151,6 +150,29 @@ def test_llama3_405B_f8(self):
         self.prepare_argv()
         self.run_and_check_perplexity()
 
+    @is_nightly
+    @pytest.mark.skip(reason="https://github.com/iree-org/iree/issues/20699")
```

Comment:
That issue is closed now. Is this the correct one or should it be: https://github.com/iree-org/iree/issues/20436